File: src/main/java/pl/yalgrin/playnite/simplesync/mapper/objects/SeriesMapper.java
package pl.yalgrin.playnite.simplesync.mapper.objects;

import org.springframework.stereotype.Component;
import pl.yalgrin.playnite.simplesync.library.domain.Series;
import pl.yalgrin.playnite.simplesync.library.dto.SeriesDTO;

@Component
public class SeriesMapper extends AbstractObjectMapper<Series, SeriesDTO> {

    @Override
    protected SeriesDTO createDTO() {
        return new SeriesDTO();
    }
}

File: src/main/kotlin/pl/yalgrin/playnite/simplesync/library/mapper/SeriesMapper.kt
package pl.yalgrin.playnite.simplesync.library.mapper

import org.springframework.stereotype.Component
import pl.yalgrin.playnite.simplesync.library.domain.Series
import pl.yalgrin.playnite.simplesync.library.dto.SeriesDTO

@Component
class SeriesMapper : LibraryObjectMapperImpl<Series, SeriesDTO>() {
    override fun createDTO(): SeriesDTO = SeriesDTO()
}